contracts/FlowCallbackScheduler.cdc

```diff
@@ -308,6 +308,67 @@ access(all) contract FlowCallbackScheduler {
         }
     }
 
+    /// SortedTimestamps maintains a sorted array of timestamps for efficient processing
```

Comment:
Most of this is pretty generic. I wish we could just put this in an ArrayUtils contract that is separate from this so we don't have all this sorting stuff cluttering up this contract, but that might not really be worth putting together